### DISCUSSION TITLE
Dynamically change a background

### DISCUSSION THREAD

**Discussion by Dmitry-kalinin** [davetcc/tcMenu#348] [Q&A] [answered]
Hello! I'm trying to dynamically override a background color. It's pretty easy to find a theme settings, but how do I change them from my code?

I'm not that experienced in C++ so excuse me if that's a trivial question.

Thnks!

**Comment by davetcc** [comment, marked as answer]
The best starting point we have at the moment documentation wise is https://www.thecoderscorner.com/products/arduino-libraries/tc-menu/themes/rendering-with-themes-icons-grids/

But in summary, if you just want to change the drawing colours, there are a couple of palette entries at the top of the theme header file, you can just change those, they are simply defined as RGB(r,g,b) where each value is between 0..255.

If you wanted to change the selected colors, you'd search for ` factory.setSelectedColors` and adjust those RGB values. That should be it.

If you wanted to change the color of a single menu, see the `esp/ESPAmplifier` example, it does that for the status menu I think.

**Reply by Dmitry-kalinin** [reply]
Thanks! For a quick fix I was able to just call a an installtheme method with a custom settings for a background. I know it's a bit of an overkill. I will invest a little bit more time into it.
